src/shogun/mathematics/linalg/LinalgNamespace.h

```diff
@@ -98,6 +98,121 @@ LinalgBackendBase* infer_backend(const Container<T>& a, const Container<T>& b)
 }
 
 /**
+ * Transfers data to GPU memory. Does nothing if no GPU backend registered.
```

Comment:
Ah I get it, so you just moved them up.
MMh, maybe this can be done in a sep patch? It kinda messes what is happening here